code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1134,3 +1134,27 @@
 		owner.update_body_parts()
 	else
 		update_icon_dropped()
+
+/obj/item/bodypart/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_WIRES || !(bodytype & BODYTYPE_ROBOTIC))
+		return FALSE
+	owner.visible_message(span_danger("[owner]'s [src.name] seems to malfunction!"))
```

Comment:
Does this print 6 simultaneous messages if you EMP someone fully augmented? Maybe no way around it, but seems a little awkward.

Reply:
Honestly with how fucked up an android gets with a full EMP its a good signifier of how crippled they're about to be.